server/src/generate_once.ts

```diff
@@ -1,6 +1,9 @@
-import { SessionManager } from "./session_manager";
+import { SessionManager, YoutubeSessionDataCaches } from "./session_manager";
 import { Command } from "@commander-js/extra-typings";
+import * as fs from "fs";
+import * as path from "path";
 
+const CACHE_PATH = path.resolve(process.cwd(), "cache.json");
```

Comment:
Can you confirm what `process.cwd()` resolves to? I'm assuming its the path of the python plugin that invokes it?

Reply:
If we want to make it agnostic of the process's current working directory, we can use a path relative to `__dirname` (the directory of the current file)

Reply:
it's where you run yt-dlp